README.md

```diff
@@ -30,48 +29,44 @@ $ pip install meeshkan
 
 Note that `meeshkan` requires **Python 3.6+.**
 
-## Python API
+## Hello world
 
-### Quick start
+The basic Meeshkan flow is **collect, build and mock.**
+1. Start by **collecting** data from recorded server traffic and OpenAPI specs.
+1. Then, **build** a schema that unifies these various data sources.
+1. Finally, use this schema to create a **mock** server of an API.
 
-```python
-from meeshkan.schemabuilder.update_mode import UpdateMode
-from meeshkan.schemabuilder import build_schema_batch, build_schema_online, update_openapi
-from openapi_typed_2 import OpenAPIObject
-import meeshkan
-from typing import List
-import json
-from http_types import HttpExchange, Request, Response, ResponseBuilder, RequestBuilder, HttpMethod
-
-
-def read_http_exchanges() -> List[HttpExchange]:
-    """Read HTTP exchanges from the source of your choice.
-    """
-    request: Request = RequestBuilder.from_url(
-        "https://example.org/v1", method=HttpMethod.GET, headers={})
-
-    response: Response = ResponseBuilder.from_dict(dict(
-        statusCode=200, body=json.dumps({"hello": "world"}), headers={}))
+```bash
+$ meeshkan record -r --daemon # start the recorder in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # record traffic
+$ meeshkan record --stop-daemon # stop the daemon
```

Comment:
Maybe `meeshkan record stop` could be easier? An alternative might also be the more Hoverfly-ish
```bash
$ meeshkan start --daemon
$ meeshkan record
$ curl ...
$ meeshkan mock
$ curl ...
$ meeshkan stop
```

Reply:
Agreed that Meeshkan record stop could be easier, I'm not too picky about the commands and their flags, so whatever Nikolay implements is fine.

I don't think we should expect the server to switch between recording and mocking on the fly at this point, as the internals of doing so is too hairy.  We have to treat them as two separate entities IMO.

Reply:
Ok, I think a single process would be simpler from user standpoint but I can see how it can get too hairy for us at this stage.

Reply:
The final syntax is 
`bash
meeshkan mock start
meeshkan mock start -d
meeshkan mock stop
meeshkan record start
meeshkan record start -d
meeshkan record stop
`